### PR TITLE
[TEST] ZD13689 - Add logic to wait until item is saved to continue

### PR DESCRIPTION
**Pull request by jdollete**

### CODE REVIEW

**Comment by hbalseiro** [comment]
It looks good and works well.

**Comment by jdollete** [comment]
@hbalseiro brought up a good point on an edge case of adding multiple questions. Will figure something out for that and re-push changes

File: www/usecases/endtoend/authoring.php [hunk 1]
```diff
@@ -126,9 +139,9 @@
             authorApp.setItem(activeItemID);
         });
         //go to assessment handler
-        $(".btn-goToAssessment").click(function(){
-            window.location = 'assessment.php?itemIDs=' + itemIDs.join(",");
-        });
+        // $(".btn-goToAssessment").click(function(){
+        //     window.location = 'assessment.php?itemIDs=' + itemIDs.join(",");
+        // });
```
Comment (hbalseiro) [inline www/usecases/endtoend/authoring.php:144]: You left commented out code here.

**Comment by sebablanco** [comment]
Is this PR still relevant or should we close it?

**Comment by jdollete** [comment]
@sebablanco , sorry about this. I will review this week. It's supposed to prevent users from going to the assessment before the item has been saved.

**Comment by jdollete** [comment]
Closing for now, will look more into it and send in another request when completed.

**Pull request closed without merging**